test/chartbasicsfactory_test.js

```diff
@@ -58,7 +60,7 @@ describe('BasicsChart', function() {
       TestUtils.renderIntoDocument(elem);
     }
     catch(e) {
-      expect(console.error.callCount).to.equal(7);
+      expect(console.error.callCount).to.equal(9);
```

Comment:
@jebeck I seem to remember you commenting once that you felt these `console.error.callCount` weren't overly useful or stable?? I might be wrong